src/wymeditor/editor/base.js

```diff
@@ -3857,14 +3855,8 @@ WYMeditor.editor.prototype._selectSingleNode = function (node) {
     selection = wym.selection();
     nodeRange = rangy.createRangyRange();
     nodeRange.selectNode(node);
-    selection.setSingleRange(nodeRange);
-};
 
-WYMeditor.editor.prototype._mouseup = function (evt) {
-    var wym = this;
-    if (evt.target.tagName.toLowerCase() === WYMeditor.IMG) {
-        wym._selectSingleNode(evt.target);
-    }
+    selection.setSingleRange(nodeRange);
 };
```

Comment:
Confusing diff here. Watch out. The poet's intention is removal of the `_mouseup` handler.

That is because we do not use it to select images any more. We use the `ImageHandler`.